pom.xml

```diff
@@ -81,6 +81,7 @@
         <module>presto-redis</module>
         <module>presto-cassandra</module>
         <module>presto-blackhole</module>
+        <module>presto-in-memory</module>
```

Comment:
`s/in-memory/inmemory`

Notice that `blackhole` is named `blackhole` not `black-hole`